src/server/protocol.ts

```diff
@@ -547,6 +551,27 @@ namespace ts.server.protocol {
         renameFilename?: string;
     }
 
+    /**

```

Comment:
This comment might be better in `organizeImports.ts` (the non-test one).

Reply:
I can move it when I implement remove-unused.